moveit_setup_assistant/src/widgets/configuration_files_widget.cpp

```diff
@@ -1118,6 +1121,18 @@ void ConfigurationFilesWidget::loadTemplateStrings()
     addTemplateString("[ROS_CONTROLLERS]", controllers.str());
   }
 
+  std::string kinematics_parameters_files_block;
+  for (const auto& groups : config_data_->group_meta_data_)
+  {
+    if (groups.second.kinematics_parameters_file_.empty())
+      continue;
+
+    std::string line = "<rosparam command=\"load\" ns=\"" + groups.first + "\" file=\"" +
```

Comment:
As far as I understand, it is possible to add multiple files here. However, only the first line is correctly indented (in the `planning_context.launch` template).
```suggestion
    std::string line = "    <rosparam command=\"load\" ns=\"" + groups.first + "\" file=\"" +
```

Reply:
Yes, one per planning group basically with the namespace (ns) being the group name

Reply:
Does this new scheme (using the `ns` attribute) still work with old configs, which just declared all kinematics parameters in one single file?

Reply:
yes, of course. You can simply leave the field for the additional kinematics parameter file on the group setting page empty and everything stays the same. But you could also choose the moveit_opw_kinematics plugin (or possibly in future UR kinematics) and an additional file with parameters distributed by ROS-I (or elsewhere). Then you would have a "old config" that just loads the correct plugin and new parameter files (from a description package) that add in the parameters.

Reply:
Thanks for your patience. I think I got the logic now.

Reply:
You're welcome! I promise to write a tutorial once we have the plugin released as well.